global.json

```diff
@@ -1,5 +0,0 @@
-{
-  "sdk": {
-    "version": "2.0.0"
-  }
-}
```

Comment:
Would like to keep this file as it signals to AppVeyor what SDK to use.

Reply:
OK, will add it back